api/handlers_e2e_test.go

```diff
@@ -522,6 +534,7 @@ func TestAccountMaxResultsLimit(t *testing.T) {
 		includeDeleted bool
 		errStatus      int
 	}{
+		{address: test.AccountA, exclude: []string{}},
```

Comment:
This test is using maxResults = 14 so you are going to hit a 400 here. (because AccountA has 20 resources)
```suggestion
		{address: test.AccountA, exclude: []string{}, errStatus: http.StatusBadRequest},
```